README.md

```diff
@@ -170,12 +171,21 @@ Immediately triggers a new search for the given text
 
 - text `String` A word of sequence of words to search for
 
-#### `search.playResult(result)`
+**Returns:**
+- retVal `Promise` - A promise that will resolve with the results of the search you just performed.
+The promise will be rejected with a timeout error if the search takes too long.
+
+#### `search.playResult(searchText, result)`
 Immediately attempts to play the given result.  If we fail to play the given result
 an error will be thrown.
 
+- searchText `String` The search text that causes the search result to appear
```

Comment:
What's wrong with implementing those 4 lines in your application? =/
On Jun 19, 2016 14:01, "Samuel Attard" <notifications@github.com> wrote:

> In README.md
> <https://github.com/gmusic-utils/gmusic-ui.js/pull/3#discussion_r67615699>
> :
>
> >  Immediately attempts to play the given result.  If we fail to play the given result
> >  an error will be thrown.
> >
> > +- searchText `String` The search text that causes the search result to appear
>
> What about putting back the hold playResult(result) method and adding a restoreSearchAndPlayResult(searchText,
> result) method.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/gmusic-utils/gmusic-ui.js/pull/3/files/9140981c7fed0e92b2a9c1ed58c5b86e4d00dcba..0910e48755e7a5c1f9d5085c45599c602f52324c#r67615699>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA3FWF_46RMveGCjbo81kbr7ghPOF2exks5qNS-DgaJpZM4I462I>
> .
>